include/tscore/Diags.h

```diff
@@ -56,7 +56,21 @@
 #endif
 #endif
 
-extern inkcoreapi Diags *diags;
+class DiagsPtr
```

Comment:
I think for core use of Dbg(), we should assert, or at least warn when there is a duplication of a control (i.e. two or more versions of a debug label).